.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 tmp
 config/local.json
-social-media
\ No newline at end of file
+social-media
```

Comment:
should this be `social-media/` so that the extra images dont get pushed ?

Reply:
Earlier you were pushing new images. now you are removing the existing images. the current tests that were dependent on these images might start failing if we merge your PR !
![image](https://user-images.githubusercontent.com/97341921/236356897-7acfebaa-c1a2-4d49-8d50-51dbc36b763a.png)

Reply:
No it won't affect the pr because they are screenshots which need not to be available for test to run. @heyrandhir